lib/wallet/test/unit/Internal/Cardano/Write/Tx/BalanceSpec.hs

```diff
@@ -2238,29 +2208,23 @@ dummyMnemonic = SomeMnemonic $ either (error . show) id
 -- Byron style addresses, corresponding to the change addresses generated by
 -- "byron wallets".
 dummyByronChangeAddressGen :: AnyChangeAddressGenWithState
-dummyByronChangeAddressGen = AnyChangeAddressGenWithState
-    (defaultChangeAddressGen @(RndState 'Mainnet) (byronRootK, pwd))
-    (mkRndState byronRootK 0)
+dummyByronChangeAddressGen = flip AnyChangeAddressGenWithState () $
+    ChangeAddressGen
+        { genChangeAddress = \() -> (dummyBootAddr, ())
+        , maxLengthChangeAddress = dummyBootAddr
+        }
   where
-    byronRootK = Byron.generateKeyFromSeed mw mempty
-    mw = SomeMnemonic $ either (error . show) id
-        (entropyToMnemonic @12 <$> mkEntropy "0000000000000000")
-    pwd = mempty
+    -- Hard-coded address originally derived from a Byron wallet using
+    -- cardano-wallet.
+    dummyBootAddr = fromJust $ deserialiseAddr $ unsafeFromHex
+        "82d818584283581c34a938c9cbd3a6f18635a29a754c3fa8cf8098b0a1c46d278ece2e75a101581e581c45c79d48621a8957dc3122167e9cb2a9651dd13d0b4b7a9a4f2a408d001adb76aa8f"
```

Comment:
Instead of introducing a new constant, why not just reuse `maxLengthAddressFor`?

See: 78f9d12a956005001620bf72d51af42f3ffc71b9

Reply:
I dropped the change to `dummyByronChangeAddressGen` for now